lib/matplotlib/axis.py

```diff
@@ -1079,15 +1079,15 @@ def _get_tick_bboxes(self, ticks, renderer):
                 [tick.label2.get_window_extent(renderer)
                  for tick in ticks if tick.label2.get_visible()])
 
-    def get_tightbbox(self, renderer, *, ignore_label=None):
+    def get_tightbbox(self, renderer, *, for_layout_only=False):
```

Comment:
I'm 50/50 on this change. Consistency is nice, OTOH, the kwarg name doesn't make a lot of sense at this level.  But since the docs refer to *_layout maybe its OK.